README.md

```diff
@@ -174,6 +176,10 @@ trainer.save_checkpoint(CKPT)
 
 Note that saving or restoring checkpoint files will stage the checkpoint file in CPU memory during save/restore, requiring additional available CPU memory equal to the size of the checkpoint file.
 
+##### Async Checkpointing
+
+Our checkpointing implementation has built-in support for the experimental [AsyncCheckpointIO](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.plugins.io.AsyncCheckpointIO.html#asynccheckpointio) featureset. This is an optimzation that allows for non-blocking `save_checkpoint` calls during a training loop. For more details on our support for this feature please see the [checkpoint README](https://github.com/GoogleCloudPlatform/dataflux-pytorch/tree/main/demo/lightning/checkpoint/singlenode#using-asynccheckpointio). This feature is supported in both single and multinode executions with the Accelerated Dataloader for Pytorch.
```

Comment:
The multinode one isn't based on AsyncCheckpointIO (if I remember correclty) and integrates with some specific pytorch distributed checkpoint interface instead. Can you update the text to be clearer about that?